CHANGES

```diff
@@ -7,6 +7,8 @@ Dependencies
 Incompatible changes
 --------------------
 
+* #7784: i18n: The msgid for alt text of image is changed
```

Comment:
This change brings a breaking change. So it might be better to merge this into 4.0.
@shimizukawa any comments?

Reply:
Good.

Reply:
Okay, rebasing now.